### PR TITLE
Update libero/coding-standard to 0.3

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: src/Adapter/FilesystemItems.php [hunk 1]
```diff
@@ -135,7 +135,7 @@ public function list(int $limit = 10, ?string $cursor = null) : ItemListPage
             $found = null;
             $i = 0;
             foreach ($ids as $id) {
-                if ($id == $cursor) {
+                if ($id->equals($cursor)) {
```
Comment (thewilkybarkid) [inline src/Adapter/FilesystemItems.php]: This is the 'downside' to https://github.com/libero/php-coding-standard/pull/20.

Comment (giorgiosironi) [reply src/Adapter/FilesystemItems.php]: Welcome to Java?

Comment (giorgiosironi) [reply src/Adapter/FilesystemItems.php]: What adds to the confusion here however is that `$cursor` changes type from `string` to `ItemId`

Comment (thewilkybarkid) [reply src/Adapter/FilesystemItems.php]: Agreed, that's not good. Will rename.

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**